### PR TITLE
Español simplistic solution using site package

### PR DESCRIPTION
**Pull request by jsuarez314**
I decided to use the site package to load tools module.

And modify the path on the notebooks to load data.

### CODE REVIEW

File: Espanol/SnowWhiteDwarf_es.ipynb [hunk 1]
```diff
@@ -503,7 +506,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "logg = pd.read_csv('dat/logg.txt', sep='\\s+', comment='#', names=['Cuerpo', 'Gravedad en superficie [g]'])\n",
+    "logg = pd.read_csv('../dat/logg.txt', sep='\\s+', comment='#', names=['Cuerpo', 'Gravedad en superficie [g]'])\n",
```
Comment (alxogm) [inline Espanol/SnowWhiteDwarf_es.ipynb:509]: Sorry, but I'm confused this is what I already see in binder using the master branch. As well as some of the other changes...

Comment (jsuarez314) [reply Espanol/SnowWhiteDwarf_es.ipynb:509]: It seems that these changes are not reflected on binder. In the Figure you can see the folders and files that I see in this folder on binder.

image

**Comment by michaelJwilson** [comment]
sorry, I take some responsibility for that:  that is in some sense by design. The Binder instances are connected to tagged releases of the repository rather than to master.  That way, there's less chance that someone commits to master and breaks the binder startups.  

You can tell as the url in the drop-in README includes the release number, and should be changed at mybinder ... after creating a new tagged release.

**Comment by alxogm** [comment]
> sorry, I take some responsibility for that: this is in some sense by design. The Binder instances are connected to tagged releases of the repository rather than to master. That way, there's less chance that someone commits to master and breaks the binder startups.

but still there is something weird, I see a different thing in binder than what John is seeing... I just started the binder from the link in the README...

**Comment by michaelJwilson** [comment]
You'd have to be using different urls?  I.e. did John pick an old one up in the slack channel, for instance. 

I checked the Spanish & English READMEs are both:
https://mybinder.org/v2/gh/michaelJwilson/DESI-HighSchool/1.2.3

which incidentally, fails to load the docker image some of the time :s

**Comment by jsuarez314** [comment]
Yes sorry! I have clicked the docker button :disappointed: 

So, in this order the unique modification that is valid for now is the Rydberg Formula.
Thanks for the replies. So, I could collaborate in this project for next experiments :grinning:

**Comment by alxogm** [comment]
> Yes sorry! I have clicked the docker button 😞
> 
> So, in this order the unique modification that is valid for now is the Rydberg Formula.
> Thanks for the replies. So, I could collaborate in this project for next experiments 😀

Thanks for looking into it @jsuarez314 !

**Pull request closed without merging**